src/utils/utils.js

```diff
@@ -31,6 +31,30 @@ export const shortenAddress = (address) => {
   );
 };
 
+/**
+ * Generate a token using the users address
+ * We need a token for a path of exile character as a proof of ownership
+ * Token is _G + last 4 letters of pool contract address + last 4 letters of user address
+ * (notice we removed the numbers from the addresses because poe character names cant use numbers)
+ * @param {String} address
+ */
+export const generateGenieToken = (address, poolAddress) => {
+  const TOKEN_LEN = 4;
+
+  let addressNoNums = address.replace(/[0-9]/g, '');
+  let poolAddressNoNums = poolAddress.replace(/[0-9]/g, '');
+
+  let token =
```

Comment:
you can do this with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals